code/gradle.properties

```diff
@@ -14,14 +14,14 @@ android.enableJetifier=true
 moduleProjectName=messaging
 moduleName=messaging
 moduleAARName=messaging-phone-release.aar
-moduleVersion=2.1.1
+moduleVersion=2.1.2
 
 mavenRepoName=AdobeMobileMessagingSdk
 mavenRepoDescription=Adobe Experience Platform Messaging extension for the Adobe Experience Platform Mobile SDK
 mavenUploadDryRunFlag=false
 
-mavenCoreVersion=2.1.0
+mavenCoreVersion=2.1.2
```

Comment:
should we bump this to 2.1.3 and wait for core release?  it contains some important fixes, especially for the race in event history